math/calculator.cpp

```diff
@@ -163,6 +167,15 @@ void requestData() {
             break;
         }
 
+        // Check user's OS
+        // Uses system("cls") if OS is Windows
+        #ifdef WINDOWS
         system("cls");
+        #endif
+
+        // Uses system("clear") if OS is Linux
+        #ifdef LINUX
```

Comment:
incorrect ifdef checks-
https://stackoverflow.com/questions/6649936/c-compiling-on-windows-and-linux-ifdef-switch

But, preferably, the "clear" command can be removed altogether. It does not add value to the problem and the code.